operator/Dockerfile

```diff
@@ -1,7 +1,8 @@
 # We have to emulate directory layout as in the repo so that imports in go files work fine.
 ARG roxpath=/workspace/src/github.com/stackrox/rox
+ARG GO_VERSION=latest
```

Comment:
I'd leave out default value to make sure that unspecified value doesn't lead to a successful build with an unpredictable go version.
```suggestion
ARG GO_VERSION
```